indico/modules/events/registration/templates/display/registration_summary.html

```diff
@@ -84,6 +84,11 @@
             {% if registration.state.name == 'complete' and registration.registration_form.tickets_enabled and
                   registration.registration_form.ticket_on_summary_page and not registration.is_ticket_blocked %}
                 <div class="group">
+                    {% if registration.registration_form.ticket_google_wallet_enabled %}
+                        <a href="{{ url_for('.ticket_google_wallet', registration.locator.registrant) }}" class="i-button accept icon-ticket" style="margin-right: 10px;">
```

Comment:
The button is missing the right border.. Maybe just get rid of the margin altogether?
![image](https://github.com/indico/indico/assets/8739637/172166b4-69bb-46fd-bc1e-93e21b338e87)